content/quickstart/cpp/tracing.md

```diff
@@ -394,62 +395,62 @@ With the above adjustments our sample alongside the updated `BUILD` and `WORKSPA
 
 ```cpp
 #include <iostream>
+
+#include "absl/strings/string_view.h"
+#include "opencensus/exporters/trace/zipkin/zipkin_exporter.h"
 #include "opencensus/trace/sampler.h"
 #include "opencensus/trace/span.h"
-#include "opencensus/exporters/trace/zipkin/zipkin_exporter.h"
-
-std::string processLine(opencensus::trace::Span &parentSpan, std::string in) {
-    opencensus::trace::Span span = \
-                    opencensus::trace::Span::StartSpan("processLine", &parentSpan);
 
-    std::string out(in);
+std::string processLine(const opencensus::trace::Span& parentSpan,
+                        const std::string& in) {
+  auto span = opencensus::trace::Span::StartSpan("processLine", &parentSpan);
+  std::string out(in);
 
-    for (auto it = out.begin(); it != out.end(); it++)
-        *it = std::toupper(*it);
+  for (auto it = out.begin(); it != out.end(); it++) *it = std::toupper(*it);
 
-    // Just add a custom annotation to examine later on.
-    span.AddAnnotation(out);
+  // Add a custom annotation to examine later on.
+  span.AddAnnotation(out);
 
-    span.End();
-    return out;
+  span.End();
+  return out;
 }
 
-int main(int argc, char **argv) {
-    // Samplers are potentially expensive to construct. Use one long-lived
-    // sampler instead of constructing one for every Span.
-    static opencensus::trace::AlwaysSampler sampler;
+int main(int argc, char** argv) {
```

Comment:
Lol at the `char** argv` vs `char **argv`

Reply:
I ran it through clang-format. :)